setup.py

```diff
@@ -59,14 +59,14 @@ def _single_compile(obj):
 
 SWIG_OPTS = ['-c++', '-python', '-O', '-shadow', '-keyword',
              '-w-511', '-w-509', '-outdir', '.']
-SWIG_PYGSL_OPTS = ['-python', '-keyword', '-outdir', '.']
+#SWIG_PYGSL_OPTS = ['-python', '-keyword', '-outdir', '.']
```

Comment:
please feel free to remove these lines that you commented out, if you decide that we don't need to ship with gsl anymore.